source/switching_to_live/set_up_a_live_stripe_account/index.html.md.erb

```diff
@@ -1,17 +1,17 @@
 ---
-title: Connect your live account to Stripe
-last_reviewed_on: 2025-03-26
+title: Connect your live service to Stripe
```

Comment:
this is sort of out of line with the idea that services aren't "live" until they've fully set up with the PSP and are able to take payments, so in that sense you can't connect a live service to Stripe, as you would have to have already connected the service to Stripe for it to be live